src/NzbDrone.Core/Localization/Core/en.json

```
@@ -1842,6 +1842,14 @@
   "SeasonFinale": "Season Finale",
   "SeasonFolder": "Season Folder",
   "SeasonFolderFormat": "Season Folder Format",
+  "SeasonImportAllowAnyWarning": "Allow a season pack if it upgrades any episode. Note that this may result in stuck queue items.",
+  "SeasonImportAllowHelpText": "Require a season pack to be a quality or custom format upgrade for all episodes",
+  "SeasonImportAllowLabel": "Allow Season Importing",
+  "SeasonImportThresholdHelpText": "Require a season pack to be an upgrade for at least X percent of episodes.",
```

```suggestion
  "SeasonPackImportThresholdHelpText": "Require a season pack to be an upgrade for at least X percent of episodes.",
```

(and same for others, this relates to SeasonPacks, not Seasons)